docker/elastic-agent/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG STACK_VERSION=8.2.0-SNAPSHOT
```

Comment:
I guess, this is something we need to bump automatically, right?

Reply:
we can, the default value is never used, this parameter is passed from the docker-compose